portal/student.py

```diff
@@ -0,0 +1,17 @@
+from flask import (
+    Blueprint, flash, g, redirect, render_template, request, session, url_for
```

Comment:
Flash isn't being used yet so it is not needed to be imported. Neither is redirect, url_for, or request. Once it is needed it can then be added so that it is not accidentally left in here unused.